recipes/modin/LICENSE

```diff
@@ -0,0 +1,201 @@
+                                 Apache License
```

Comment:
A license header is included in every source file in the package, so I'm not sure about whether we need to manually package the license file. Then the question would be how to specify the `license_file` below. I'm not sure best how to handle this.